positron/branding/unofficial/locales/en-US/brand.dtd

```diff
@@ -0,0 +1,8 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<!ENTITY  brandShortName  "Positron">
+<!ENTITY  brandFullName   "Mozilla Positron">
+<!ENTITY  vendorShortName "Mozilla">
+<!ENTITY  logoTrademark   "">
```

Comment:
Yeah, I suppose this one could be removed... It's only shown on the about popup it seems, so it's not being read from code. It's probably clearer to not define it then.